src/settings/settings.ts

```diff
@@ -45,6 +45,10 @@ export const appearanceSettings = createGlobalSettings<Appearance>('apperance',
     primary: () => i18n.t('settings_appearance'),
 })
 
+export const responsiveUISettings = createGlobalSettings<boolean>('responsiveUI', webpackEnv.target === 'WKWebview', {
```

Comment:
Why add a setting for it? It should be responsive by default 🤔

Reply:
I changed my mind media queries are better than this manner.